cockpit_api/src/main/java/com/cockpit/api/controller/TeamController.java

```diff
@@ -51,8 +51,19 @@ public ResponseEntity createTeam(@PathVariable Long mvpId,
     @GetMapping(value = "/api/v1/team/all")
     public ResponseEntity getTeams(@RequestHeader("Authorization") String authHeader) {
         if (authService.isUserAuthorized(authHeader)) {
-            List<TeamDTO> mvpList = teamService.findAll();
-            return ResponseEntity.ok(mvpList);
+            List<TeamDTO> teamList = teamService.findAll();
+            return ResponseEntity.ok(teamList);
+        } else {
+            return ResponseEntity.ok(HttpStatus.UNAUTHORIZED);
+        }
+    }
+
+    // GET ALL MEMBERS
+    @GetMapping(value = "/api/v1/teamMember/all")
+    public ResponseEntity getAllMembers(@RequestHeader("Authorization") String authHeader) {
```

Comment:
ResponseEntity<Object> in order to remove sonar code smell

Reply:
For all old controllers, this action has been done in the branch of code smell fixing

Reply:
done